packages/myst-cli/src/process/file.ts

```diff
@@ -108,6 +108,11 @@ export function loadTexFile(
   return { kind: SourceFileKind.Article, mdast: tex.ast as GenericParent, frontmatter };
 }
 
+export function loadMySTJSON(content: string) {
+  const { mdast, frontmatter } = JSON.parse(content);
+  return { kind: SourceFileKind.Article, mdast, frontmatter };
```

Comment:
Should we also allow `kind` to be pulled from the content? (but still default back to `Article`)

Reply:
Ooh, I like that idea. Feel free to tweak, or I'll merge.

Reply:
Tweaked!

Reply:
MERGE AT WILL ✈️✈️